openupgrade_scripts/scripts/point_of_sale/14.0.1.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,103 @@
+---Models in module 'point_of_sale'---
+---Fields in module 'point_of_sale'---
+point_of_sale / pos.config               / iface_precompute_cash (boolean): DEL
+point_of_sale / pos.config               / manage_orders (boolean)       : NEW
```

Comment:
I think we have to enable this by default on existing configs, for providing this new feature, specially if coming from `pos_order_mgmt`. We are working on the transition from this module for this version (it seems the return is not available, but the config says that it exists). cc @chienandalu

Reply:
Yes, it says it exists but it doesn't. For point_of_sale, v14 is an advance of the functionallity that landed in v15. Indeed, this field doesn't exist in that version anymore, as they backed away from this approach (and in v16 the UI flow is similar although the whole module got a refactor.

In a summary I'd be setting it on by default as it's going to be in future versions.